DevREADME.md

```diff
@@ -2,27 +2,17 @@
 
 AutoCAD Lisp Extension is a vscode extension for debug AutoCAD AutoLISP. The Extension plays the roles of both debug adapter and language server which could enable you debug lisp with AutoCAD. The following description is for developers.
 
-## How to setup the Dev Env
-npm install -g yarn (if it's not installed yet)
+## How to setup the Dev env and compile the code
 
-Then you could setup the develop environment by:
-```
-cd AutoLispExt
-npm install --global gulp-cli
-npm install 
-```
-__NOTE:__ You may see a warning `"The engine "vscode" appears to be invalid"` while running `yarn install`. This is a known problem. Please refer to the [VSCode issue](https://github.com/microsoft/vscode/issues/91009).
-## How to compile the code
-You could compile the code by:
+You could do all above steps  by the script pack.py, it is python2
 ```
 cd AutoLispExt
-gulp build
-copy utils\acadProcessFinder\bin\acadProcessFinder.exe extension\out\process
+python pack.py
 ```
-If you only want to build the ts file and debug it via F5, you can run:
+### How to compile the codes
+The script pack.py will copy some utility files to correct location for making package. After run this script and then change some ts codes, you can also use the follow command to compile it:
 ```
-=> Windows:   winCompile.bat
-=> Otherwise: ./macCompile.sh
```

Comment:
this two files also clean up the out folder, so it's a clean build in this way, which is different from npm run compile

Reply:
Maybe this is two detail, I think it is best to hide the details for new developer of this project. The only thing now is to run the pack.py script.

Reply:
It would be nice if we could get this setup where F5 handles the out cleaning, compile, file copy and debug. I didn't handle the "cleaning the folder" aspect in my FabCOD extension, but I was able to bind a python script to copy my dependency files that send a script to AutoCAD using COM.
https://github.com/AgileBIM/FabCOD/blob/main/.vscode/tasks.json

Reply:
Thanks for pointing out this ideal!